typescript-selenium/TestConstants.ts

```diff
@@ -0,0 +1,37 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+function readEnvAndSetValue(envProperty: any, defaultValue: string | number | boolean): any {
+    const propertyValue = envProperty
+    if (!propertyValue) {
+        return defaultValue;
+    }
+
+    return propertyValue
+}
+
+
+
+export const TestConstants = {
+    TS_SELENIUM_HEADLESS: readEnvAndSetValue(process.env.TS_SELENIUM_HEADLESS, false),
```

Comment:
This code is unnecessarily complicated. You can replace it with something like this for boolean values
```js
TS_SELENIUM_HEADLESS: process.env.TS_SELENIUM_HEADLESS === 'true',
```
this for numbers
```js
TS_SELENIUM_DEFAULT_ATTEMPTS: Number(process.env.TS_SELENIUM_DEFAULT_ATTEMPTS) || 5,
```
or this for strings
```js
TS_SELENIUM_BASE_URL: process.env.TS_SELENIUM_BASE_URL || "http://che-che.192.168.99.100.nip.io"
```

Reply:
Nice catch! Thank you.